Default.sublime-commands

```diff
@@ -77,6 +77,10 @@
         "caption": "git: graph",
         "command": "gs_log_graph"
     },
+    {
+        "caption": "git: graph current branch",
+        "command": "gs_log_graph_current_branch",
```

Comment:
the build failed. May be caused by the extra commas?

Reply:
Maybe, I'll check